test_elements.py

```diff
@@ -22,25 +21,29 @@ def test_update_movie(self):
   def test_new_series(self):
     serie = Serie('Breaking Bad', 'Vince Gilligan', 2008)
     self.assertEqual(serie.title, 'Breaking Bad')
-    self.assertEqual(serie.director, 'Vince Gilligan')
+    self.assertEqual(serie.creator, 'Vince Gilligan')
     self.assertEqual(serie.year, 2008)
 
   def test_update_serie(self):
-    movie = Serie('Peaky Blinders', 'Steven Knight', 2013)
+    serie = Serie('Peaky Blinders', 'Steven Knight', 2013)
     serie.update('Breaking Bad', 'Vince Gilligan', 2008)
     self.assertEqual(serie.title, 'Breaking Bad')
-    self.assertEqual(serie.director, 'Vince Gilligan')
+    self.assertEqual(serie.creator, 'Vince Gilligan')
     self.assertEqual(serie.year, 2008)
   
   def test_new_song(self):
     song = Song('Lose Yourself', 'Eminem', 2002)
-    self.assertEqual(serie.title, 'Lose Yourself')
-    self.assertEqual(serie.director, 'Eminem')
-    self.assertEqual(serie.year, 2002)
+    self.assertEqual(song.title, 'Lose Yourself')
+    self.assertEqual(song.artist, 'Eminem')
+    self.assertEqual(song.year, 2002)
   
   def test_update_song(self):
     song = Song('Money Trees', 'Kendrick Lamar', 2012)
     song.update('Lose Yourself', 'Eminem', 2002)
-    self.assertEqual(serie.title, 'Lose Yourself')
-    self.assertEqual(serie.director, 'Eminem')
-    self.assertEqual(serie.year, 2002)
\ No newline at end of file
+    self.assertEqual(song.title, 'Lose Yourself')
+    self.assertEqual(song.artist, 'Eminem')
```

Comment:
should it be creator instead of artist here? since director was changed to creator above